docs/src/api/params.md

```diff
@@ -605,6 +605,13 @@ for a list of supported timezone IDs. Defaults to the system timezone.
   - `accuracy` ?<[float]> Non-negative accuracy value. Defaults to `0`.
 
 ## context-option-locale
+* langs: js
+- `locale` <[string]>
+
+Specify user locale, for example `en-GB`, `de-DE`, etc. Locale will affect `navigator.language` value, `Accept-Language` request header value as well as number and date formatting rules. Defaults to `en-US`. Learn more about emulation in our [emulation guide](../emulation.md#locale--timezone).
+
+## context-option-locale-java-csharp-python
```

Comment:
This would require you to reference it in params.md: https://github.com/search?q=repo%3Amicrosoft%2Fplaywright%20%2Fcontext-option-locale%2F&type=code